pkg/gofr/container/container.go

```diff
@@ -75,8 +75,17 @@ func (c *Container) Create(conf config.Config) {
 	}
 
 	if c.Logger == nil {
+		levelFetchConfig, err := strconv.Atoi(conf.GetOrDefault("REMOTE_LOG_FETCH_INTERVAL", "15"))
+		if err != nil {
+			levelFetchConfig = 15
+		}
+
 		c.Logger = remotelogger.New(logging.GetLevelFromString(conf.Get("LOG_LEVEL")), conf.Get("REMOTE_LOG_URL"),
-			conf.GetOrDefault("REMOTE_LOG_FETCH_INTERVAL", "15"))
+			time.Duration(levelFetchConfig)*time.Second)
+
+		if err != nil {
+			c.Logger.Error("incorrect value for REMOTE_LOG_FETCH_INTERVAL. setting default of 15 sec.")
```

Comment:
we are using `incorrect` here and `invalid` in the request_timeout log.. pls use the same for consistency in logs.

Reply:
sure corrected